content/integrations/jira-integration.md

```diff
@@ -0,0 +1,88 @@
+
+3. Create a [Jira API Token](https://id.atlassian.com/manage-profile/security/api-tokens) once you have access to Jira.
+
+## Add the codemagic.yaml to your project
+
+Copy the **codemagic.yaml** from the starter project into the root of your repository. Update the environment variables as indicated and use the documentation links where required for the values related to building your app, code signing and app publishing. 
+
+## Update the Jira environment variables
+
+There are four **environment variables** that need to be updated for the Jira integration: `JIRA_AUTH` `JIRA_BASE_URL` `JIRA_ISSUE` and `JIRA_TRANSITION_ID`
+
+
+#### Updating the JIRA_AUTH environment variable
+
+The `JIRA_AUTH` environment variable is a base64 encoded string which consists of the email address you log into Jira with and the Jira API token you created: 
+
+`email@example.com:<api_token>`
+
+You can encode these credentials in the **macOS Terminal** using:
+
+```
+echo -n 'email@example.com:<api_token>' | openssl base64
+```
+
+Alternatively use an online tool to base64 encode this string. 
+
+This value is used in the Authorization header used in cURL requests to the Jira API.
+
+#### Adding the JIRA_BASE_URL environment variable
+
+This is the subdomain you chose when you set up your Jira account e.g. "YOUR_SUBDOMAIN.atlassian.net". Put the subdomain including 'atalassian.net' in the `JIRA_BASE_URL` environment variable. 
+
+
+#### Specifying the  the JIRA_ISSUE environment variable
+
+Issues, epics and stories have a unique id which is usually in the format **'XXXX-N'** and is visible on your issues either in the bottom right or top left when looking at an issue. Put this value in the `JIRA_ISSUE` environment variable. 
+
+#### Finding and updating the JIRA_TRANSITION_ID environment variable
+
+If you want to transition your issue to another status you will need to know what transition ids are available. You can obtain the available transition ids using a cURL request as documented in the [Jira API documentation](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issues/#api-rest-api-3-issue-issueidorkey-transitions-get). Once you know the transition id then put this value in the `JIRA_TRANSITION_ID` environment variable.
+
+## Adding formatted comments to a Jira issue
+
+Copy the **.scripts** folder to the root of your project. This folder contains a template file called **jira.json** which is used to add formatted comments to a Jira issue. 
+
+The Atlassasion Document Format (ADF) is used to format the comment layout and style. Click [here](https://developer.atlassian.com/cloud/jira/platform/apis/document/structure/) for more information about ADF and how to modify this template. 
+
+![A formatted Jira issue comment](../uploads/jira_issue_comment.png)
+
+**Note** that it contains strings beginning with $ which the scripts use to replace values in the json using sed before it is added as json payload to the cURL requests.
```

Comment:
I would format the file formats, symbols, commands, tools with with backtics or in bold for easier reading